README.md

```diff
@@ -94,8 +103,56 @@ Now you are ready to test out the app locally! To do this, simply run `npm run d
 
 Now you have played with the AI tamago locally -- it's time to deploy it somewhere more permanent so you can access it anytime! 
 
-While I'm still writing up a detailed deployment guide, here are the resources you can refer to for deploying different parts of the app:  
+**0. Choose which model you want to use in production**
+- If you want to test out using Chatgpt in prod, simply remove `LLM_MODEL=ollama` from `.env.local` and fill in `OPENAI_API_KEY`
+- If you want to try [Replicate](https://replicate.com/), set `LLM_MODEL=replicate_llama` and fill in `REPLICATE_API_TOKEN`
+- If you want to deploy Ollama yourself, you can follow this awesome guide -- [Scaling Large Language Models to zero with Ollama](https://fly.io/blog/scaling-llm-ollama/). It is possible to run Ollama on a `performance-4x` Fly VM (CPU) with `100gb` volume, but if you can get access to GPUs they are much faster. Join Fly's GPU waitlist [here](https://fly.io/gpu) if you don't yet have access!
+
+
+**1. Switch to `deploy` branch -- this branch includes everything you need to deploy an app like [this](https://ai-tamago.fly.dev/).**
+
+   
+```git co deploy```
```

Comment:
**suggestion (documentation):** Consider using full 'git checkout' command for better compatibility

Some users might not have git aliases configured. Using 'git checkout deploy' would ensure the command works for everyone.

```suggestion
```git checkout deploy```
```